Homework1

TestKartashov opened the pull request:
Первая домашка

TestKartashov committed:
Первый ДЗ

TestKartashov committed:
gitignore

TestKartashov committed:
Первая домашка

TestKartashov committed:
Первая домашка

TestKartashov committed:
Первая домашка

TestKartashov committed:
Первая домашка

TestKartashov committed:
Первая домашка

luchanos commented on less1/dz6.py:
```diff
@@ -0,0 +1,34 @@
+try:
+    a = int(a_txt)
+    b = int(b_txt)
+except:
```
а что отлавливаем?)

TestKartashov replied:
если пользователь введет буквенное значение, сделал общий Exception

luchanos commented on less1/dz6.py:
```diff
@@ -0,0 +1,34 @@
+    b = int(b_txt)
+except:
+    print('Неверное значение')
+    exit()
```
гасим интерпретатор, зачем так ультимативно?)

TestKartashov replied:
а как, если значения кривые, и дальнейшее прохождение не нужно

luchanos commented on less1/dz6.py:
```diff
@@ -0,0 +1,34 @@
+while a < b:
+    a = a + ((old_day * 10) / 100)
+    old_day = a
+    # print("%d-й день: %.2f" % (cnt, a))
```
это нам нужно?) не оставляй под комментарием код в чистовом варианте

TestKartashov replied:
ок

luchanos commented on less1/dz5.py:
```diff
@@ -0,0 +1,42 @@
+
+# Определите, с каким финансовым результатом работает фирма (прибыль — выручка больше издержек, или убыток — издержки больше выручки).
+if revenue == cost:
+    print("The company worked without loss and without profit")
```
+++

luchanos commented on less1/dz4.py:
```diff
@@ -0,0 +1,34 @@
+num = 0
+while i < len(n_str):
+    if len(n_str) > (i + 1) and num < int(n_str[i + 1]):
+        num = int(n_str[i + 1])
```
+++

luchanos replied:
ещё подумай над тем, чтобы при натыкании на девятку сразу завершать поиск) сэкономит операции)

TestKartashov replied:
Принял. Спасибо. Изменил

luchanos commented on less1/dz2.py:
```diff
@@ -0,0 +1,31 @@
+min = (second % 3600) / 60
+sec = (second % 3600 % 60)
+
+print('чч:мм:сс %02d:%02d:%02d' % (hour, min, sec))
```
+++

luchanos commented on less1/dz1.py:
```diff
@@ -0,0 +1,31 @@
+# выведите на экран.
+
+
+var_1 = 'test'
```
+++

TestKartashov committed:
Первая домашка
ещё подумай над тем, чтобы при натыкании на девятку сразу завершать поиск) сэкономит операции)

TestKartashov committed:
Первая домашка
ещё подумай над тем, чтобы при натыкании на девятку сразу завершать поиск) сэкономит операции)

TestKartashov committed:
Исправил Exception

TestKartashov committed:
Исправил Exception
